Sming/SmingCore/SmingCore.h

```diff
@@ -55,7 +55,8 @@
 #include "Data/Stream/FileStream.h"
```

Comment:
You not' only change the content of the file but also its permissions. `Sming/SmingCore/SmingCore.h 100644 → 100755`. The file permission change has to be reverted.

Reply:
Not sure what happened there. Cygwin seems to do odd things at times. Fixed in working copy. Will push when coding style checked and committed.

Reply:
Fixed in last push.